### DISCUSSION TITLE
Where is the details for next step?

### DISCUSSION THREAD

**Discussion by eliassal** [skills/.github#761] [Code with Copilot]
Hi I followed instructions at https://learn.microsoft.com/en-us/training/modules/introduction-to-github-copilot/5-exercise, I have VSC has the copilt up and ready. After forking the repo, I see a phrase

> Wait about 60 seconds then refresh your repository landing page for the next step.

I refreshed the repo so many times but I dont see any detilas for next step.
Thanks for your help
![image](https://github.com/user-attachments/assets/817b4ad5-4c37-425f-b260-8f03d1e7e0ec)

**Comment by AaronChristianHernandez** [comment]
Any updates?
